documentation/src/main/markdown/integrations/acr/azurecontainerregistry.md

```diff
@@ -0,0 +1,105 @@
+
+**[solution_name] - Scanning Images**   
+```
+bash <(curl -s -L https:‎ //detect.synopsys.com/detect8.sh) \
+--blackduck.url=<URL> \
+--blackduck.api.token=<token> \
+--detect.docker.image=<Image URI> \
+--detect.project.name=<Project Name>
+```
+
+## Invoking [solution_name] as a script to scan a Docker image stored in ACR
+
+If you would rather run [solution_name] as a script than an extension, follow these steps:
+
+In this example, follow the steps to create your first application using the [Azure Portal.](https://docs.microsoft.com/en-us/azure/devops/pipelines/get-started-azure-devops-project?view=vsts)   
+ 
+From the available options, select: **Node.js sample app > Simple Node.js app > Web App for Containers.**   
+ 
+You must authenticate with ACR; refer to [Authenticate with Azure Container Registry](https://docs.microsoft.com/en-us/azure/container-registry/container-registry-authentication).
+
+Start in **Pipelines > Library** inside Azure DevOps. 
+
+1\. Refer to [Variable Groups for Builds and Releases](https://docs.microsoft.com/en-us/azure/devops/pipelines/library/variable-groups?view=vsts) page for how to create a Variable Group. 
+
+2\. Create a variable group for your [blackduck_product_name] instance:
+
+- blackduck.url (value is the url of your [blackduck_product_name] instance).
+
+- blackduck.api.token (value is your generated API token, secret).
+
+3\. Create a second variable group for your ACR Credentials:
+- acr.username (value is your ACR username).
+
+- acr.password (value is your ACR password).
+
+4\. Access your build(CI) pipeline by expanding the **Pipelines** sidebar item, and then choosing **Builds**.  
+
+5\. Select the Pipeline you want to add [solution_name] to, then click **Edit**.   
+
+6\. Link your variable groups by following the steps in [Use a Variable Group](https://docs.microsoft.com/en-us/azure/devops/pipelines/library/variable-groups?view=vsts#use-a-variable-group).  
+ 
+7\. Add a Pipeline task for running [solution_name]:   
+- After you click **Edit**, the **Tasks** screen of your CI Pipeline opens.
+- In the **Build** task, click the plus (**+**) sign to add a new task.
+- Use the search bar to search for bash.
+- Click **Add** to add the step to your pipeline.
+
+8\. Configure the bash step to run after the image has been pushed to ACR.   
+- Select to run an inline script.
+- Reference the following example for the script to run [solution_name].   
+
+```
+#/bin/bash
+#Log in to ACR using the configured Variable Group
+docker login <registryname>.azurecr.io -u $(acr.username) -p $(acr.password)
+#Call [solution_name], passing the Docker Image location
+bash <(curl -s -L https:‎ //detect.synopsys.com/detect8.sh) \
+--blackduck.url=$(blackduck.url) \
+--blackduck.api.token=$(blackduck.api.token) \ --detect.docker.image=<registryname>.azurecr.io/<containername>:$(Build.BuildId) \ --detect.project.name=$(Build.DefinitionName) \ 
```

Comment:
Is this formatted correctly? Usually there is a carriage return after the "\"

Reply:
No and yes. Fixed! Thank you.